server/middlewares/schema.js

```diff
@@ -245,3 +245,14 @@ export const resetEmailSchema = {
     .email({ minDomainSegments: 2 })
     .required()
 };
+
+export const searchFilterSchema = {
+  searchQuery: Joi.string().trim().min(2),
```

Comment:
Nice job with this Monday. I think with the way you created your schema, we can abstract some already pre-existing schema to avoid duplication. Just like the `email` and `password` fields from the `sign-up` and `login` routes.

Reply:
Noted. I didn't think about it that way. I will look into it. Thanks for the observation